aula_2/CarlosAmarante/exercicio4.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime, date
+
+
+def is_greater_than_18(birthday):
+    current_time = datetime.now()
+    age = current_time.year - birthday.year - (
+        (current_time.month, current_time.day) < (birthday.month, birthday.day)
+        )
+    if age >= 18:
```

Comment:
you can use `return age >= 18`.

Reply:
I really need to fix this in my mind hahaha